core/src/main/java/org/infinispan/xsite/XSiteAdminOperations.java

```diff
@@ -280,7 +280,7 @@ public final String pushState(@Parameter(description = "The destination site nam
       try {
          stateTransferManager.startPushState(siteName);
       } catch (Throwable throwable) {
-         log.debugf(throwable, "Unable to pushState to '%s'.", siteName);
+         log.errorf(throwable, "Unable to pushState to '%s'.", siteName);
```

Comment:
@pruivo I'd rather keep it the same as the error string that's returned to the user.